Resources/Prototypes/Entities/Objects/Misc/bouquet.yml

```diff
@@ -0,0 +1,16 @@
+- type: entity
+  name: PoppyBouquet
```

Comment:
```suggestion
  name: flower bouquet
```

Reply:
considering afaict you can make this with other flowers too?

Reply:
tag is Flower so yes, all flowers are available